ctlearn/run_model.py

```diff
@@ -506,10 +358,66 @@ def write(prediction):
         '--log_to_file',
         action='store_true',
         help="log to a file in model directory instead of terminal")
+    parser.add_argument(
```

Comment:
Following an offline discussion with @TjarkMiener and @LucaRomanato, the `multiple_runs` command line argument should be removed entirely. It doesn't work well in practice, and the same functionality can be obtained if desired using `run_multiple_configurations.py`. Can you take it out (and update the Readme to match)?

Reply:
Done